assets/src/stories-editor/blocks/amp-story-page/edit.js

```diff
@@ -302,6 +305,12 @@ class PageEdit extends Component {
 											VIDEO_ALLOWED_MEGABYTES_PER_SECOND
 										)
 									}
+									{
+										videoBytesPerSecond > 0 && ' ' + sprintf(
+											__( 'The selected video is %d MB per second.', 'amp' ),
```

Comment:
Missing translator comment

Reply:
Fixed in 1d20f42bc7bcb9d7e4adfca1c1ef2ffd60b4b86b